gcp/example-usage

```diff
@@ -0,0 +1,80 @@
+# Example usage for this terraform-bastion module.
+# This example does not use all possible module inputs,
+# please see [README.md](./README.md) for descriptions of all inputs.
+
+module "bastion" {
+  # Replace the `ref` below, with a `gcp-vxxx` release from https://github.com/FairwindsOps/terraform-bastion/releases
+  source = "git@github.com:FairwindsOps/terraform-bastion.git//gcp?ref=gcp-v0.1.0"
+
+  bastion_name       = "kube-bastion"
+  region             = "us-central1"
+  availability_zones = ["us-central1-a", "us-central1-b", "us-central1-c"]
+
+  # This can also be set in the shell environment, using TF_VAR_infrastructure_bucket
+  infrastructure_bucket = "CompanyName-kube-infrastructure"
+
+  # THis allows this module to use the same GCS bucket for multiple invocations.
+  infrastructure_bucket_bastion_key = "default/bastion"
+
+  dns_zone_name = "kube-company-com"
+
+  # This network houses the subnetwork below it.
+  network_name    = "kube"
+  subnetwork_name = "kube-nodes"
+
+  # This uses an SSH public key from the Terraform root module directory.
+  ssh_public_key_file                = file("${path.root}/bastion_ssh.pub")
```

Comment:
using the `file` function as shown here clashes with the `file` function in `instance-template.tf`. It worked if I dropped the function here, so the example should probably be updated to `ssh_public_key_file                = "${path.root}/../config/private/${var.ssh_key}"`

Reply:
I gave this input an unfortunate name in the AWS module - I've updated the GCP module to match the AWS one, and updated the input descriptions to clarify how the input is used despite its name. The input represents the content, not the file name, of the SSH public key. I opened [this issue](https://github.com/FairwindsOps/terraform-bastion/issues/25) to consider renaming the input.

Reply:
Ah ok, thanks for clarifying!